apps/admin/frontend/src/components/import_cvrfiles_modal.test.tsx

```diff
@@ -355,115 +299,24 @@ describe('Screens display properly when USB is mounted', () => {
       _batchLabel: 'Batch 1',
     };
 
-    const mockFiles = CastVoteRecordFiles.empty;
-
-    // Add initial "live" CVR file to lock the file mode:
-    const added = await mockFiles.addAll(
-      [new File([JSON.stringify(cvr)], 'randomname')],
-      eitherNeitherElectionDefinition.election
-    );
-
-    window.kiosk!.readFile = jest.fn().mockImplementation((path) =>
-      JSON.stringify({
-        ...cvr,
-        _testBallot: path !== 'live1',
-      })
-    );
-
-    const backend = new ElectionManagerStoreMemoryBackend({
-      electionDefinition: eitherNeitherElectionDefinition,
-      castVoteRecordFiles: added,
+    mockApiClient.getCastVoteRecordFileMode
+      .expectCallWith()
+      .resolves(Admin.CvrFileMode.Official);
+    mockApiClient.getCastVoteRecordFiles
+      .expectCallWith()
+      .resolves([{ ...mockCastVoteRecordFileRecord, filename: 'random' }]);
+    renderInAppContext(<ImportCvrFilesModal onClose={jest.fn()} />, {
+      usbDrive: mockUsbDrive('mounted'),
+      apiClient: mockApiClient,
     });
-    const { getByText, getAllByTestId } = renderInAppContext(
-      <ImportCvrFilesModal onClose={closeFn} />,
-      {
-        usbDrive: mockUsbDrive('mounted'),
-        backend,
-      }
-    );
     await screen.findByText('Load Official Ballot Mode CVR Files');
 
-    const tableRows = getAllByTestId('table-row');
+    const tableRows = screen.getAllByTestId('table-row');
     expect(tableRows).toHaveLength(1);
     domGetByText(tableRows[0], '12/09/2020 03:59:32 PM');
     domGetByText(tableRows[0], cvr._scannerId);
-    jest.spyOn(backend, 'addCastVoteRecordFile').mockResolvedValueOnce({
-      wasExistingFile: false,
-      newlyAdded: 0,
-      alreadyPresent: 0,
-      exportedTimestamp: new Date().toISOString(),
-      fileMode: Admin.CvrFileMode.Test,
-      fileName: 'cvrs.jsonl',
-      id: 'cvr-file-1',
-      scannerIds: ['scanner-2', 'scanner-3'],
-    });
     expect(
       domGetByText(tableRows[0], 'Load').closest('button')!.disabled
     ).toEqual(false);
-
-    fireEvent.click(getByText('Cancel'));
-    expect(closeFn).toHaveBeenCalledTimes(1);
-
-    fireEvent.click(domGetByText(tableRows[0], 'Load'));
-    await screen.findByText('Loading');
-    await waitFor(() => {
-      expect(backend.addCastVoteRecordFile).toHaveBeenCalledTimes(1);
-      getByText('0 new CVRs Loaded');
-    });
-  });
-
-  test('Shows previously loaded files when all files have already been loaded', async () => {
```

Comment:
lots of duplicate coverage of this file across tests, tried to consolidate into more targeted tests.